community/open-source.md

```diff
@@ -5,3 +5,14 @@ Example:
 * [AwesomeLibrary](): descriptive text, consider additional links to docs or
```

Comment:
@justinyoo thanks for this. Can you do me a favor and remove the template bullet (the "AwesomeLibrary" thing), just to save me the trouble of creating another PR to remove it. Now that we have one (yours) we can use that as the template for others to follow.

Reply:
@duglin Sure thing. Done.